rosbag2_storage/include/rosbag2_storage/topic_metadata.hpp

```diff
@@ -25,6 +25,8 @@ struct TopicMetadata
   std::string name;
   std::string type;
   std::string serialization_format;
+  // std::vector<rclcpp::QoS> serialized to YAML string
```

Comment:
(very minor) I thought it was commented code! Maybe rephrase to add an empty line before the comment to help the reader?

Reply:
I'll rephrase the comment, I can see how that would be confusing at a glance

Reply:
I'll rephrase the comment, I can see how that would be confusing at a glance